internal/generator/vector/elements/elements.go

```diff
@@ -31,10 +31,12 @@ route.{{$route_name}} = {{$route_expr}}
 }
 
 type Remap struct {
-	ComponentID string
-	Desc        string
-	Inputs      string
-	VRL         string
+	ComponentID    string
+	Desc           string
+	Inputs         string
+	GroupBy        string
+	LogGroupPrefix string
```

Comment:
strongly suggest not to change the basic elements for a specific purpose.
vector Remap does not have these fields https://vector.dev/docs/reference/configuration/transforms/remap/